process/throttle/antiflood/factory/p2pOutputAntiflood.go

```diff
@@ -28,6 +29,9 @@ func initP2POutputAntiFlood(mainConfig config.Config) (process.P2PAntifloodHandl
 		return nil, err
 	}
 
+	//since this is the output antiflooder, we will provide a dummy self peer id in order to not count twice the values
+	dummyPeerId := core.PeerID("dummy")
```

Comment:
would not use dummy in name - add something like unique ?

Reply:
removed dummy, left it as plain "peerId", kept the comments explaining why it should be like this